tests/virt_autotest/install_package.pm

```diff
@@ -16,27 +16,28 @@ use base "virt_autotest_base";
 use testapi;
 use virt_utils;
 use utils;
+use repo_tools 'add_qa_head_repo';
+use Utils::Architectures 'is_s390x';
 
-sub install_package {
-
-    my $qa_server_repo = get_var('QA_HEAD_REPO', '');
-    if ($qa_server_repo eq '') {
-        #default repo according to version if not set from testsuite
-        $qa_server_repo = 'http://dist.nue.suse.com/ibs/QA:/Head/SLE-' . get_var('VERSION');
-        set_var('QA_HEAD_REPO', $qa_server_repo);
-        bmwqemu::save_vars();
-    }
-    if (check_var('ARCH', 's390x')) {
-        lpar_cmd("zypper --non-interactive rr server-repo");
```

Comment:
I am afraid 'zypper rr' needs to be kept, or it will cause trouble in second triggering, especially s390, which reuses host. So does x86 for potentia

Reply:
Understand, I add `zypper_wrapper("rr $repo")` for both archs.